textproc/libxml2/Portfile

```diff
@@ -57,6 +57,7 @@ configure.args      --disable-silent-rules \
 destroot.keepdirs   ${destroot}${prefix}/etc/xml
 post-destroot {
     xinstall -m 0755 -d ${destroot}${prefix}/etc/xml
+    ln -s ../unicode ${destroot}${prefix}/include/libxml2/unicode
```

Comment:
What directory is `../unicode`? Can the full path be specified?

Reply:
It isn't in `${destroot}` it is from `port:icu`.

Reply:
So `${prefix}/include` right? I think the link should be specified that way for clarity. 

`ln -s ${prefix}/include/unicode ${destroot}${prefix}/include/libxml2/unicode`

Reply:
done